packages/playground/App.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Sample React Native App
```

Comment:
I missed that we were using a source copy from React Native here ☹️ 

The template is actually published as part of the standard RN package. Could we link to this in node_modules instead? I'm guessing the version we have of this is already outdated.